src/index.html

```diff
@@ -26,5 +26,19 @@
     <script src="assets/js/dropdown.js"></script>
     <script src="bitflux.js"></script>
     <script src="app.js"></script>
+    <script>
+        (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
+                    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
+                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+
+        ga('create', 'UA-75463920-1', 'auto');
+        ga('send', 'pageview');
+
+        var reportAction = function(action) {
+            ga('set', 'page', '/index.html#'+action);
+            ga('send', 'pageview');
```

Comment:
I'd probably say it's worth it, so that it's not necessarily conditioned on the variable being previously defined at a certain point on the code.